src/app/store/AuthModel.ts

```
@@ -554,104 +674,42 @@ export const getAuthModel = (): AuthModel => ({
             const res = await resultGravityAccessToken.json()
-            showError(res, reject, "facebook")
+            showError(res, reject, "google")
           }
         }
-      }
-
-      // get info from facebook
-      const infoRequest = new GraphRequest(
-        "/me",
-        {
-          accessToken: accessToken.accessToken,
-          parameters: {
-            fields: {
-              string: "email,name",
-            },
-          },
-        },
-        responseFacebookInfoCallback
-      )
-      new GraphRequestManager().addRequest(infoRequest).start()
-    })
-  }),
-  authGoogle: thunk(async (actions, options) => {
-    return await new Promise<AuthPromiseResolveType>(async (resolve, reject) => {
-      if (!(await GoogleSignin.hasPlayServices())) {
-        reject(new AuthError("Play services are not available."))
-      }
-      const userInfo = await GoogleSignin.signIn()
-      const accessToken = (await GoogleSignin.getTokens()).accessToken
-
-      if (options.signInOrUp === "signUp") {
-        const resultGravitySignUp = userInfo.user.name
-          ? await actions.signUp({
-              email: userInfo.user.email,
-              name: userInfo.user.name,
-              accessToken,
-              oauthProvider: "google",
-              agreedToReceiveEmails: options.agreedToReceiveEmails,
-            })
-          : { success: false, message: "missing name in google's userInfo" }
-        resultGravitySignUp.success
-          ? resolve({ success: true })
-          : reject(
+      } catch (e) {
+        if (e instanceof Error) {
+          if (e.message === "DEVELOPER_ERROR") {
+            reject(
               new AuthError(
-                resultGravitySignUp.message,
-                resultGravitySignUp.error,
-                resultGravitySignUp.meta
+                "Google auth does not work in firebase beta, try again in a playstore beta",
+                e.message
               )
             )
```

google auth is not working in firebase betas and right now it is just failing silently, let's show devs an error message otherwise it is unclear what is happening